xrpl/models/requests/request.py

```diff
@@ -74,6 +74,9 @@ class RequestMethod(str, Enum):
     PING = "ping"
     RANDOM = "random"
 
+    # generic unknown/unsupported request
+    UNKNOWN_REQUEST = "unknown_request"
```

Comment:
can we add to this comment to say that it has no analogue on xrpl, IE it's only an xrpl-py thing?